### ISSUE TITLE
SSD tensorflow checkpoint inconsistent with the reference model

### ISSUE THREAD

**Issue by wang12tao** [mlperf/inference_policies#7]
SSD 1200 tensorflow checkpoint: https://zenodo.org/record/3247091#.XQ0S4pJKhGE is inconsistent with the reference model: https://github.com/mlperf/inference/blob/master/v0.5/classification_and_detection/python/models/ssd_r34.py#L295

Batchnorm is added after each convolution in additional layers in the checkpoint, for example, ssd1200/additional_layers/conv8/conv8_bn1/beta is found in the checkpoint.

But in the reference model, there is no batchnorm added after convolution: https://github.com/mlperf/inference/blob/master/v0.5/classification_and_detection/python/models/ssd_r34.py#L295

**Comment by wang12tao** [comment]
In the checkpoint's additional_layers, there is no bias for convolutions, but bias=True in the reference code: https://github.com/mlperf/inference/blob/master/v0.5/classification_and_detection/python/models/ssd_r34.py#L295

**Comment by wang12tao** [comment]
There is bias in the checkpoint's convolution layers of Resnet-34, but there is no bias in the reference code: https://github.com/pytorch/vision/blob/ec203153095ad3d2e79fbf2865d80fe6076618fa/torchvision/models/resnet.py#L23

**Comment by christ1ne** [comment]
@jveejay to ask tao and xilinx folks to align on the model in the next week.

**Comment by jveejay** [comment]
@christ1ne I think this is the latest resolution for this issue: https://zenodo.org/record/3262269 -- as in they updated the model and the checkpoint together so they should be aligned.

Can you verify and close?

**Comment by christ1ne** [comment]
I think the issue can be closed as the latest model is running fine. Please re-open the issue if needed.

**Comment by wang12tao** [comment]
I checked the new checkpoint https://zenodo.org/record/3262269, model architecture is still inconsistent with Pytorch reference.

Could SSD tensorflow owner share the SSD-large tensorflow training code with us? Thanks!

**Comment by jveejay** [comment]
Adding ashish to make sure he can help us get in touch with the right folks at Xilinx.

Typed on a small screen, excuse brevity and typos.
vj.

________________________________
